unit_testing/myapp/models.py

```diff
@@ -6,5 +7,27 @@ class Person(models.Model):
     first_name = models.TextField(max_length=50)
     last_name = models.TextField(max_length=50)
 
+    cache_key = u'person_{}'
+
     def __unicode__(self):
         return "{name}".format(name=self.first_name+self.last_name)
+
+    def save(self, *args, **kwargs):
+        super(Person, self).save(*args, **kwargs)
```

Comment:
The cache should also be cleared on saving the object. Otherwise it will not reflect the changes on update.